src/main/resources/rp-start

```diff
@@ -21,4 +21,40 @@ if [ "$RP_DYN_JAVA_OPTS" != "" ]; then
   export JAVA_OPTS="$(eval "echo $RP_DYN_JAVA_OPTS") $JAVA_OPTS"
 fi
 
+# For DC/OS, Marathon, Mesos, port env names are static. Tooling expects
+# them to be of a particular name, so it sets the value to the name of
+# the assigned port env var. We then have to evaluate them to resolve
+# their values.
+#
+# Example: RP_ENDPOINT_AKKAREMOTE_BIND_HOST='$HOST' will be resolved
+#          to the value of $HOST
+
+if [ "$RP_PLATFORM" = mesos ]; then
+    ENDPOINT_SUFFIXES="HOST,PORT,BIND_HOST,BIND_PORT"
```

Comment:
Alternatively, we could have the CLI provide this. Perhaps less coupled, slightly more complex.